frontend/src/container/Tracim.jsx

```diff
@@ -156,6 +163,21 @@ export class Tracim extends React.Component {
     if (fetchGetContentTypeList.status === 200) props.dispatch(setContentTypeList(fetchGetContentTypeList.json))
   }
 
+  loadLiveMessage = async () => {
+    const { props } = this
+    const response = await props.dispatch(getLiveMessage())
+
+    switch (response.status) {
+      case 200:
+        console.log('got live messages already sent')
+        break
+      default:
+        // props.dispatch(newFlashMessage(('Error while getting live messages'), 'error'))
```

Comment:
It is because since the feature isn't ready yet, I avoid showing a flash message for something that will fail 100%.

I will fix it in later release realted to live message